### PR TITLE
fix(auction): Fix image resolution

### PR DESCRIPTION
**Pull request by damassi**
Fixes the image resolution on the main header image, and other weirdness with the bid modal image / active bids.
 
 

Screen Shot 2022-03-15 at 12 40 19 PM

### CODE REVIEW

File: src/v2/Apps/Auction/AuctionApp.tsx [hunk 1]
```diff
@@ -113,9 +113,7 @@ export const AuctionAppFragmentContainer = createFragmentContainer(AuctionApp, {
 
       internalID
       coverImage {
-        cropped(width: 1800, height: 600, version: "normalized") {
-          src
-        }
+        url
```
Comment (dzucconi) [inline src/v2/Apps/Auction/AuctionApp.tsx:116]: This should be fine but I'd probably figure out what versions this image has and then specify 2 or 3 in size order

Comment (damassi) [reply src/v2/Apps/Auction/AuctionApp.tsx]: fixed 👍

File: src/v2/Apps/Auction/AuctionApp.tsx [hunk 2]
```diff
@@ -113,9 +113,7 @@ export const AuctionAppFragmentContainer = createFragmentContainer(AuctionApp, {
 
       internalID
       coverImage {
-        cropped(width: 1800, height: 600, version: "normalized") {
-          src
-        }
+        url(version: ["larger", "large"])
```
Comment (dzucconi) [inline src/v2/Apps/Auction/AuctionApp.tsx:116]: So I checked Gemini to see what versions we actually have:

```
gemini:staging> Template.where(key: 'sale').first.versions.map { |x| [x.key, x.width, x.height] }
=> [["large_rectangle", 640, 480], ["wide", 2000, nil], ["square", 230, 230], ["source", 1920, 1920]]
```

So looks like you want this to be `version: ["wide", "source", "large_rectangle"]`

Comment (dzucconi) [reply src/v2/Apps/Auction/AuctionApp.tsx:116]: (Also it's worth noting that none of these are really adequate; we'd probably want to add a normalized to this.)

Comment (damassi) [reply src/v2/Apps/Auction/AuctionApp.tsx]: Updated 👍

**Pull request merged**